Package.swift

```diff
@@ -56,14 +80,131 @@ let package = Package(
+                "Generated/AutoCases.generated.swift",
+                "Generated/AutoEquatable.generated.swift",
+                "Generated/AutoHashable.generated.swift",
+                "Generated/AutoLenses.generated.swift",
+                "Generated/AutoMockable.generated.swift",
+                "Generated/LinuxMain.generated.swift",
+                "Expected",
+                "Info.plist",
+                "TemplatesTests.swift"
+            ],
+            sources: [
+                "Context/AutoCodable.swift",
+                "Generated/AutoCodable.generated.swift"
+            ]
+        ),
+        .target(name: "TryCatch", path: "TryCatch", exclude: ["Info.plist"]),
+        .testTarget(
+            name: "SourceryLibTests",
+            dependencies: [
+                "SourceryLib",
+                "Quick",
+                "Nimble"
+            ],
+            exclude: [
+                "Info.plist"
+            ],
+            resources: [
+                .copy("Stub/Configs"),
+                .copy("Stub/Errors"),
+                .copy("Stub/JavaScriptTemplates"),
+                .copy("Stub/SwiftTemplates"),
+                .copy("Stub/Performance-Code"),
+                .copy("Stub/Result"),
+                .copy("Stub/Templates"),
+                .copy("Stub/Source")
+            ]
+        ),
+        .testTarget(
+            name: "CodableContextTests",
+            dependencies: [
+                "CodableContext",
+                "Quick",
+                "Nimble"
+            ],
+            path: "Templates/CodableContextTests",
+            exclude: [
+                "Info.plist"
+            ]
+        ),
+        .testTarget(
+            name: "TemplatesTests",
+            dependencies: [
+                "Quick",
+                "Nimble",
+                "PathKit"
+            ],
+            path: "Templates",
+            exclude: [
+                "CodableContext",
+                "CodableContextTests",
+                "Tests/Generated",
+                "Tests/Info.plist",
+                "default.profraw"
+            ],
+            sources: [
+                // LinuxMain is not compiled as part of the target
+                // since there is no way to run script before compilation begins.
+                "Tests/TemplatesTests.swift"
+            ],
+            resources: [
+                .copy("Templates"),
+                .copy("Tests/Context"),
+                .copy("Tests/Expected")
+            ]
+        )
     ]
 )
+
+hookInternalSwiftSyntaxParser()
+
+/// We need to manually add an -rpath to the project so the tests can run via Xcode
+/// If we are running from console (swift build & friend) we don't need to do it
+func hookInternalSwiftSyntaxParser() {
+    let isFromTerminal = ProcessInfo.processInfo.environment.values.contains("/usr/bin/swift")
+    if !isFromTerminal {
+        package
+            .targets
+            .filter(\.isTest)
```

Comment:
`-rpath` to the bundled version of SwiftSyntax should be added to the `Sourcery` target too. Otherwise it won't find the library when launched during `TemplatesTests`.
Strangely, the same executable runs normally when launched from terminal.